Add Theia.cloud monitor (#107)
eclipsesource/theia-cloud: node/monitor/README.md

# Monitor

> VSCode extension to monitor Theia.cloud hosted VSCode/Theia products. The extension will expose a rest service, where an external service can ping the extension to get information about the pod.

Reviewer: ```suggestion
VSCode extension to monitor Theia.cloud hosted VSCode/Theia products.
The extension exposes a rest service that an external service can ping to get information about the pod.
```

> The extension is designed to allow other modules to be added in the future.

Reviewer: ```suggestion
The extension is designed to facilitate the addition of further modules in the future.
```

> For this the extension will read out environment variables to check if a specific module should be activated.

Reviewer: ```suggestion
For this, the extension reads environment variables to check the activation of each corresponding module.
```

> For an example of how these environment variables should be set, take a look at `Run extension` launch config the `.vscode/launch.json` file.

Reviewer: ```suggestion
For an example on setting these environment variables, take a look at the `Run extension` launch config in the `.vscode/launch.json` file.
```
